source/documentation/doc-Release_Notes/topics/ref-ovirt-4.5.2.adoc

```diff
@@ -0,0 +1,102 @@
+=== Red Hat Virtualization 4.4 SP 1 Batch Update 2 (ovirt-4.5.2)
+
+
+
+==== Bug Fix
+
+These bugs were fixed in this release of Red Hat Virtualization:
+
+BZ#link:https://bugzilla.redhat.com/1955388[1955388]::
+Previously, the Manager could start a virtual machine with a Resize and Pin NUMA policy on a host that does not have an equal number of physical sockets to the number of NUMA nodes it has. As a result, the wrong pinning was assigned to the policy. With this release, the Manager does not allow the virtual machine to be scheduled on such a virtual machine, making the pinning correct based on the algorithm.
+
+BZ#link:https://bugzilla.redhat.com/2097558[2097558]::
+Previously, running engine-setup did not necessarily renew OVN certificates when they were close to expiration or expired. With this release, OVN certificates are always renewed by engine-setup when needed.
+
+BZ#link:https://bugzilla.redhat.com/2097725[2097725]::
+Previously, the Manager issued warnings about approaching certificate expiration before engine-setup could update the certificates. In this release the expiration warning and certificate update periods are aligned, and certificates are updated as soon as the warnings about their upcoming expiration occur.
+
+BZ#link:https://bugzilla.redhat.com/2101481[2101481]::
+The handling of core dumps during upgrade from previous Red Hat Virtualization versions to RHV 4.4 SP1 batch 1, has been fixed.
+
+BZ#link:https://bugzilla.redhat.com/2104115[2104115]::
+Previously, when importing a virtual machine with manual CPU pinning (pinned to a dedicated host), the manual pinning string was cleared, but the CPU pinning policy was not set to NONE. As a result, importing failed.
+In this release, the CPU pinning policy is set to NONE if the CPU pinning string is cleared, and importing succeeds.
+
+BZ#link:https://bugzilla.redhat.com/2105781[2105781]::
+The hosted-engine-ha binaries have been moved from /usr/share to /usr/libexec. As a result, the hosted-engine --clean-metadata command fails. With this release, 
+you must use the new path for the command to succeed: /usr/libexec/ovirt-hosted-engine-ha/ovirt-ha-agent
+
+BZ#link:https://bugzilla.redhat.com/2109923[2109923]::
+Previously, it was not possible to import templates from the Administration Portal.
+With this release, importing templates from the  Administration Portal is now possible.
+
+==== Enhancements
+
+This release of Red Hat Virtualization features the following enhancements:
+
+BZ#link:https://bugzilla.redhat.com/1793207[1793207]::
+A new warning has been added to the vdsm-tool to protect users from using the unsupported user_friendly_names multipath configuration. The following is an example of the output:
+
+$ vdsm-tool is-configured --module multipath
```

Comment:
This is not rendering correctly because it is not marked as a code block.

Reply:
we don't use code blocks in release notes just plain text

Reply:
It is totally unreadable because Asciidoctor renders it as a continuous paragraph with no breaks. The user can't possibly figure out how to update the multipath configuration if it looks like this:

![image](https://user-images.githubusercontent.com/31725355/189170583-8ef63133-55a7-442a-ae58-c428fceeaee4.png)